app/src/main/kotlin/com/wire/android/ui/home/conversations/details/editselfdeletingmessages/EditSelfDeletingMessagesViewModel.kt

```diff
@@ -117,13 +118,19 @@ class EditSelfDeletingMessagesViewModel @Inject constructor(
             val currentSelectedDuration = state.locallySelected
             state = when (updateMessageTimer(conversationId, currentSelectedDuration!!.value.inWholeMilliseconds)) {
                 is UpdateMessageTimerUseCase.Result.Failure -> {
+                    appLogger.e("Failed to update self deleting enforced duration for conversation=${conversationId.toLogString()} " +
+                            "with new duration=${currentSelectedDuration.name}")
                     state.copy(isLoading = true)
                 }
 
-                UpdateMessageTimerUseCase.Result.Success -> state.copy(
-                    isLoading = false,
-                    remotelySelected = currentSelectedDuration
-                )
+                UpdateMessageTimerUseCase.Result.Success -> {
+                    appLogger.d("Success updating self deleting enforced duration for conversation=${conversationId.toLogString()} " +
```

Comment:
maybe we can log the old duration ?

Reply:
also I think we need to have this stuff in json format otherwise we will not be able to pick the individual info with the datadog

Reply:
forget it It is done on Kaliuim side 🤦